internal/daemonset/mock_daemonset.go

```diff
@@ -12,9 +12,9 @@ import (
 	context "context"
 	reflect "reflect"
 
-	v10 "github.com/openshift/cluster-nfd-operator/api/v1"
+	v1 "github.com/openshift/cluster-nfd-operator/api/v1"
 	gomock "go.uber.org/mock/gomock"
-	v1 "k8s.io/api/apps/v1"
+	v10 "k8s.io/api/apps/v1"
```

Comment:
the same for the v1 / v10 swap here, I dont see that it makes much difference to the functionality, so fine, but its not clear to me how it relates to scc's

Reply:
This is the result of running generate command...